ubiquity/dell-bootstrap.py

```diff
@@ -627,6 +641,11 @@ def fixup_factory_devices(self, rec_part):
         #EFI install finds ESP
         self.db.set('grub-installer/bootdev', self.device)
 
+        # install GRUB if it's dmraid installation
+        if self.device.startswith("/dev/dm"):
+            grub_command = 'debconf-set partman-auto/disk "$(ls /dev/mapper/isw* | head -n1)"'
```

Comment:
Don't you have an actual mapping to device ID using `transfer_dmraid_path`?  Why not use that here instead?
This would then prevent potential failures from situations that have multiple `/dev/dm` or multiple `/dev/mapper/isw` prefixes.

Reply:
That's a good suggestion.